laterpay/asset_sources/js/laterpay-backend-account.js

```diff
@@ -13,14 +14,20 @@
                 isLive                      : 'lp_is-live',
 
                 // plugin mode
+                pluginModeIndicator         : $('#lp_js_pluginModeIndicator'),
                 pluginModeToggle            : $('#lp_js_togglePluginMode'),
+                pluginVisibilitySetting     : $('#lp_js_pluginVisibilitySetting'),
+                pluginVisibilityToggle      : $('#lp_js_toggleVisibilityInTestMode'),
+                pluginModeTestText          : $('#lp_js_pluginMode_testText'),
+                pluginModeLiveText          : $('#lp_js_pluginMode_liveText'),
+                    // FIXME: wtf is this??
```

Comment:
And we need that to switch from "visible" to "invisible", if the credentials are invalid? (which would be very good behavior)